sections/futures/Trade/Trade.tsx

```diff
@@ -106,6 +107,14 @@ const Trade: React.FC = () => {
 		}
 	}, [positionLeverage, positionSide, leverageSide, marketMaxLeverage]);
 
+	const maxMarketValueUSD = marketLimitQuery?.data ?? wei(0);
+	const marketSize = market?.marketSize ?? wei(0);
+
+	const isMarketCapReached = useMemo(
+		() => marketSize?.mul(marketAssetRate ?? wei(0)).gte(maxMarketValueUSD),
```

Comment:
I don't think this logic is correct. This will compare `maxMarketValueUSD` against the total open interest for this futures market. Markets are closed when single-sided OI reaches `maxMarketValueUSD`.

Context:
<img width="538" alt="image" src="https://user-images.githubusercontent.com/10401554/165816369-2cb25b85-020d-4d28-ba86-5c071294c0a0.png">

Reply:
Thanks for the clarification.